.prettierrc

```diff
@@ -1,6 +1,6 @@
 {
   "semi": true,
   "singleQuote": true,
-  "tabWidth": 2,
+  "tabWidth": 4,
```

Comment:
Tabs should ideally be used too, as per V's styleguide.

Reply:
agree with you @Alhadis

Reply:
Tabs, because they're better than spaces. 😉

Scientifically-proven fact.